docs/user/rollout-policies.md

```diff
@@ -0,0 +1,107 @@
+# Rollout Device Selection and Rollout Disruption Budget
+
+## Overview
+
+When performing a rollout using `flightctl`, it is crucial to manage which devices participate in the rollout and how much disruption is acceptable. This document outlines the device selection process and the concept of the rollout disruption budget to ensure controlled and predictable rollouts.
+
+---
+
+## Rollout Device Selection
+
+### **Device Targeting**
+
+A rollout applies only to devices that belong to a fleet. Each device can belong to only a single fleet. Since rollout definitions are done at the fleet level, the selection process determines which devices within a fleet will participate in the rollout based on label criteria.
+
+- **Labels**: Devices with specific metadata labels can be targeted for rollouts.
+- **Fleet Membership**: Rollouts apply only to devices within the specified fleet.
+
+### **Device Selection Strategy**
+
+Currently, Flightctl supports only the **BatchSequence** strategy for device selection. This strategy defines a stepwise rollout process where devices are added in batches based on specific criteria.
+
+Batches are executed sequentially. After each batch completes, execution proceeds to the next batch only if the success rate of the previous batch meets or exceeds the configured **success threshold**. The success rate is determined as:
+
+```text
+# of successful rollouts in the batch / # of devices in the batch >= success threshold
+```
+
+A **successful rollout** is when a device completes the rollout process successfully.
+
+### **Limit in Device Selection**
+
+Each batch in the **BatchSequence** strategy uses a `limit` parameter to define how many devices should be included in the batch. The limit can be specified in two ways:
+
+- **Absolute number**: A fixed number of devices to be selected.
+- **Percentage**: A percentage of the total device population that matches the label criteria.
+  - If a `selector` with labels is provided, the percentage is calculated based on the number of devices that match the label criteria within the fleet.
+  - If no `selector` is provided, the percentage is applied to all devices in the fleet.
+
+### **Success Threshold**
+
+The **success threshold** defines the percentage of successfully updated devices required to continue the rollout. If the success rate falls below this threshold, the rollout may be paused or rolled back to prevent further failures.
```

Comment:
What do you mean by "rolled back"?  We don't have rollback.

Reply:
Right